.devcontainer/docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: "3.8"
 
 services:
   app:
-    image: node:19-bullseye
+    image: node:20-bullseye
```

Comment:
are you able to successfully build with this change? I wasn't able to as I'm aware of the badengine warning.

Reply:
on this note, I would get a lot of cache 'misses' and queries to npm registry would fail if I switched the node image. Not sure why. Could try it again today in case it was a fluke or something in my setup.

Reply:
It runs with "npm run backend:dev" and "npm run frontend:dev". What else should I do to try it?

Reply:
Also "npm run backend" and "npm run frontend" without the :dev worked.

Reply:
can you show your build logs for this?

Reply:
The output isn't very informative:

```
root@b9eb82a43330:/workspaces/LibreChat# npm run reinstall

> LibreChat@0.6.1 reinstall
> node config/update.js -l -g

Starting update script, this may take a minute or two depending on your system and network.
Deleting node_modules in /workspaces/LibreChat
Deleting node_modules in /workspaces/LibreChat/packages/data-provider
Deleting node_modules in /workspaces/LibreChat/api
Cleaning npm cache...
npm WARN using --force Recommended protections disabled.
Installing dependencies...

> LibreChat@0.6.1 prepare
> node config/prepare.js

stdout: husky - Git hooks installed

stderr: 

added 1905 packages, and audited 1909 packages in 14s

425 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
Building frontend...

> LibreChat@0.6.1 frontend
> npm run build:data-provider && cd client && npm run build


> LibreChat@0.6.1 build:data-provider
> cd packages/data-provider && npm run build


> librechat-data-provider@0.3.1 build
> npm run clean && rollup -c --silent --bundleConfigAsCjs


> librechat-data-provider@0.3.1 clean
> rimraf dist


> @librechat/frontend@0.6.1 build
> cross-env NODE_ENV=production vite build

vite v5.0.7 building for production...
✓ 2374 modules transformed.
dist/index.html                     0.81 kB │ gzip:   0.39 kB
dist/assets/index-Hfga6ewA.css    114.68 kB │ gzip:  18.45 kB
dist/assets/index--j5BzNk6.js     634.72 kB │ gzip: 175.78 kB
dist/assets/vendor-BylGBZfT.js  1,614.09 kB │ gzip: 489.04 kB

(!) Some chunks are larger than 500 kB after minification. Consider:
- Using dynamic import() to code-split the application
- Use build.rollupOptions.output.manualChunks to improve chunking: https://rollupjs.org/configuration-options/#output-manualchunks
- Adjust chunk size limit for this warning via build.chunkSizeWarningLimit.
✓ built in 6.36s
Your LibreChat app is now up to date! Start the app with the following command:
npm run backend
Note: it's also recommended to clear your browser cookies and localStorage for LibreChat to assure a fully clean installation.
Also: Don't worry, your data is safe :)
root@b9eb82a43330:/workspaces/LibreChat# npm run backend

> LibreChat@0.6.1 backend
> cross-env NODE_ENV=production node api/server/index.js

Connected to MongoDB
There are 75 messages in the database, 75 indexed
There are 4 convos in the database, 4 indexed
Server listening on all interfaces at port 3080. Use http://localhost:3080 to access it
```